tests/script_warnings/match_imut_multiple_default/warning.txt

```diff
@@ -1,7 +1,7 @@
-Warning: 
+Warning(behaviour): 
     1 | merge event of match true of
     2 |   case true => true
-    3 |   default => true
-    4 |   default => false
+    3 |   case _ => true
+    4 |   case _ => false
     5 | end end
-      | ^^^ A match statement with more then one default clause will never reach any but the first default clause.
+      | ^^^ A match statement with more then one default clause will never reach any but the first default clause.
```

Comment:
this should be an error about unreachable code.

Reply:
But we can do this in another PR, as it was there before already

Reply:
nah it's a good point I'll make it nicer!